### ISSUE TITLE
Remove white space around the chatpane (webview)

### ISSUE THREAD

**Issue by djmgit** [fossasia/susi_desktop#83]
**Actual Behaviour**

Presently there is a white border around the webview.

**Expected Behaviour**

The border should not be there, the top bar should take up the entire width.

**Screenshots of the issue**
![susi](https://user-images.githubusercontent.com/16368427/32128202-a16af56a-bb99-11e7-9f67-b7c9ac90831d.png)

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by anshumanv** [comment]
Yeah, the whitespace needs to be trimmed down.
Please send a PR. :smile:
